utils/cliutils/commandsflags.go

```diff
@@ -1387,6 +1389,10 @@ var flagsMap = map[string]cli.Flag{
 		Name:  ProxyKey,
 		Usage: "[Optional] The key of an HTTP proxy configuration in Artifactory. This proxy will be used for the transfer traffic between the source and target instances. To configure this proxy, go to \"Proxies | Configuration | Proxy Configuration\" in the JFrog Administration UI.` `",
 	},
+	transferFilesStatus: cli.BoolFlag{
+		Name:  Status,
+		Usage: "[Default: false] Set to true to show the status of a running transfer-files command.` `",
```

Comment:
"[Default: false] Set to true to show the status of a running transfer-files command.` `",
-->
"[Default: false] Set to true to show the status of the transfer-files command currently in progress.` `",